src/tasks/ruuvi_task_led.c

```diff
@@ -185,6 +185,11 @@ void rt_led_blink_isr (void * const p_context)
     active = !active;
 }
 
+void rt_led_blink_once_isr (void * const p_context)
```

Comment:
This funktion should not be visible outside this file? Declare it as 
-> `static void rt_led_blink_once_isr (void * const p_context)`

Reply:
If this function is going to be unit tested, ->
```
#ifndef CEEDLING
static
#endif
void rt_led_blink_once_isr (void * const p_context)
```

It would also be possible to define TESTABLE_STATIC as 
```
#ifdef CEEDLING
#define TESTABLE_STATIC
#else
#define TESTABLE_STATIC static
#endif
```
And then declare function as `TESTABLE_STATIC void rt_led_blink_once_isr (void * const p_context)`